demo/web/firebase_element/app_element.html

```diff
@@ -0,0 +1,39 @@
+<dom-module id="app-element">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <div class="horizontal center-justified layout">
+      <div class="vertical-section">
+        <h3>Dinosaurs by Height</h3>
+        <template is="dom-bind">
+          <firebase-collection
+              order-by-child="height"
+              location="https://dinosaur-facts.firebaseio.com/dinosaurs"
+              data="{{dinosaurs}}"></firebase-collection>
+          <template is="dom-repeat" items="[[dinosaurs]]" as="dinosaur">
```

Comment:
Mit freundlichen Grüßen

Günter Zöchbauer
guenter@gzoechbauer.com
+43 (699) 10 18 87 15

On Fri, Sep 25, 2015 at 8:16 PM, Jacob MacDonald <notifications@github.com>
wrote:

> In demo/web/firebase_element/app_element.html
> <https://github.com/dart-lang/polymer_elements/pull/63#discussion_r40459576>
> :
>
> > +  <template>
> > +    <style>
> > +      :host {
> > +        display: block;
> > +      }
> > +    </style>
> > +
> > +    <div class="horizontal center-justified layout">
> > +      <div class="vertical-section">
> > +        <h3>Dinosaurs by Height</h3>
> > +        <template is="dom-bind">
> > +          <firebase-collection
> > +              order-by-child="height"
> > +              location="https://dinosaur-facts.firebaseio.com/dinosaurs"
> > +              data="{{dinosaurs}}"></firebase-collection>
> > +          <template is="dom-repeat" items="[[dinosaurs]]" as="dinosaur">
>
> So if we modify polymer_elements/lib/src/paper-styles/demo-pages.html it
> will just get bashed over the next time we update, which I would like to
> avoid. You could for now just have a custom demo-pages.html file in the
> demo project though. Really this change needs to be made upstream, but as
> it would presumably break a number of JS demos that might be an issue.
>
> This is mostly what I did. I called it `demo-elements` because it's for
elements instead of pages, not sure if this is a good idea. At least you
can have a look how it worked out.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dart-lang/polymer_elements/pull/63/files#r40459576>.
>